frontend/src/store/app.js

```diff
@@ -36,7 +36,7 @@ export const useAppStore = defineStore('app', {
     setPageTitle(pageTitle) {
       this.pageTitle = pageTitle;
     },
-    setSubtitleBanner(state, subtitleBanner) {
+    setSubtitleBanner(subtitleBanner) {
```

Comment:
I was going to suggest dropping the setter entirely but this store is full of them so we can leave as is for now.